manifest.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Conversation rewards",
+  "description": "Generate rewards for on topic conversation for closing issues as complete.",
+  "ubiquity:listeners": [ "issues.closed" ]
```

Comment:
Sorry to keep nitpicking. I just want it to be clear as possible for new developers. We can always rename these later later in the future so it's by no means a priority or urgent to address this comment. 

What about `"ubiquity:watch"`?

Reply:
`"ubiquity:watch"` works for me. However, this is still non-standard and Chrome will complain. I am not sure what we could achieve with actual web extensions.